app/api/entities/api_key.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Entities
+  class ApiKey < Grape::Entity
```

Comment:
@maksim-litvinov Hey. We should **always** follow convention for naming. API is acronym so it cant be named like Api or ApI.